test/e2e/kubernetes/pod/pod.go

```diff
@@ -538,7 +538,7 @@ func WaitOnSucceeded(podPrefix, namespace string, sleep, duration time.Duration)
 
 // WaitOnReady will call the static method WaitOnReady passing in p.Metadata.Name and p.Metadata.Namespace
 func (p *Pod) WaitOnReady(sleep, duration time.Duration) (bool, error) {
-	return WaitOnReady(p.Metadata.Name, p.Metadata.Namespace, 2, sleep, duration)
+	return WaitOnReady(p.Metadata.Name, p.Metadata.Namespace, 6, sleep, duration)
```

Comment:
is this 6 another "default" for PodsReadinessChecks? It's confusing that some places call `pod.WaitOnReady(5*time.Second, 3*time.Minute)` while others call `pod.WaitOnReady(windowsDeploymentName, "default", 3, 30*time.Second, cfg.Timeout)` where one `pod` is the pkg while the other is the var of type *Pod

Reply:
not related to this PR, I just read the code to understand what the 6 was doing and found it confusing

Reply:
It is confusing. Tech debt that I didn't want to address here, and for the cases that include the readiness checks as a static "black box" value, I thought it was best to more tolerant vs less tolerant.